site/source/pages/api/show.hbs

```diff
@@ -16,9 +16,11 @@ Renders a cedar chart object into the specified element in the DOM.
 | `renderer`  | `string` | no | `svg` or `canvas` (default) for drawing the chart. |
 | `width`     | `number` | no | Explicit pixel width of chart. If not specified, will use width of `elementId`. |
 | `height` 	  | `number` | no | Explicit pixel height of chart. If not specified, will use height of `elementId`. |
+| `maxLabelLength` | `number` | no | Explicit max length of axis labels. If set, axis labels will be truncated to the given length. |
+| `callback` | `function` | no | Optional callback to better capture errors from requests and vega.
```

Comment:
`data` Would contain EITHER the response from the request from the server OR the spec being passed to the chart constructor. (yes this is `bad` that it's two entirely idfferent things. IMO)